apps/site/src/views/Resources/components/HeadingCard/HeadingCard.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as utils;
+
+$mobile-item-padding-y: 26px;
+$mobile-item-padding-x: 32px;
+
+$item-padding-y: 32px;
+$item-padding-x: 48px;
+
+$heading-font-size: 28px;
```

Comment:
Suggestion: we prefer using relative units such as rem for font sizes to account for different browser font sizes settings.

Reply:
On my end it still says 28px